index.js

```diff
@@ -0,0 +1,32 @@
+const taskList = () => {
+  console.log(`
+  ===================
+  現在持っているのタスク
+  ===================
+  `);
+};
+taskList();
+
+let currentTask = [
+  ['机を片付ける', '掃除'],
+  ['牛乳を買う', '買い物'],
+  ['散歩する', '運動']
+];
+
+const taskDisplay = () => { 
+  for (let i = 0; i < currentTask.length; i ++) {
+    console.log(`${i} : [内容] ${currentTask[i][0]},[ジャンル] ${currentTask[i][1]}`);
+  };
+};
+taskDisplay();
+
+const newTask = prompt('タスクを入力してください');
+const newGenre = prompt('ジャンルを入力してください');
+currentTask.push([newTask, newGenre]);
```

Comment:
Object の配列に Object を追加する場合は以下のようにします。

```suggestion
currentTask.push({ content: newTask, genre: newGenre });
```